kpi/tests/api/test_api_tos.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.models import User
+from django.utils.timezone import now
+from django.urls import reverse
+from rest_framework import status
+
+from kpi.tests.base_test_case import BaseTestCase
+
+
+class TOSTestCase(BaseTestCase):
+    def setUp(self) -> None:
+        self.url = reverse(self._get_endpoint('tos'))
+        self.user = User.objects.create_user(
+            username='someuser',
+            password='someuser',
+            is_active=True,
+        )
+        self.client.login(username='someuser', password='someuser')
+
+    def test_post(self):
+        # Prepare and send the request with empty payload
+        time = now().strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
Might want to consider declaring this string: `'%Y-%m-%dT%H:%M:%SZ` as a constant as it is used at least 10 times in the repository.